.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Set default charset and indentation for pyton files
+[*.py]
+charset = utf-8
+indent_style = space
+indent_size = 4
+
+# 2 space indentation for yml fies and package.json
+[{package.json,*.yml}]
+indent_style = space
```

Comment:
Set `trim_trailing_whitespace` as true too, don't set this in `[*]` because some file like md and rst have intentional trailing space.